contrib/python/conan_helpers.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
Comments on Python files: 
- Encoding hint at the beginning is missing (usually # -*- coding: utf-8 -*-): https://www.python.org/dev/peps/pep-0263/
- I'd suggest to use `"""Comment"""` instead of `# Comment` for non-inline comments such as the module comment on top as this is the official docstring style (https://www.python.org/dev/peps/pep-0257/)
- For our license comment, we could use __license__ and __author__

Which version of Python are we using, btw? For Python3, we could consider type hints... but since this is all basically helper code for our pipeline, it's probably not worth the effort, just a suggestion.

I know this is not the first python file we add to the repo, so this is probably the case for others as well. Are you following a styleguide here?

Reply:
Regarding the python version: It's python-3 but might a pretty old version based on the docker container. From the top of my head I don't know if all containers are on 3.5. That's why I haven't looked into type-hints so far. In general I like this suggestion very much. :smiley: 

And we don't have a styleguide, neither for python, nor for shell or powershell. We should probably introduce some rules for all of them.

Reply:
> For our license comment, we could use license and author

Can you explain that? Is there some standard pythonic way for license headers? Or do I miss something completely here? :see_no_evil:

Reply:
I thought this was standard, but apparently it's more of a best practice that some projects are following. The idea is to put this kind of information in module-level dunder variables: https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files

But since it's not standard, just including our default copyright notice should be fine, I'd just change it to be included in triple quotes.

Reply:
Ahh. Got it. Thanks. I have to admit I haven't seen this before. :see_no_evil: 
And as far as I remember we are advised to use the current license header (which was provided by the Google OpenSource team if I'm not mistaken.).

I will change the comments now! Thanks a lot!

Reply:
By the way I'm planning on applying your suggestions on the whole `conanfile.py`. But there are changes coming up from me (in another branch) and from Cris (due to OrbitlClientGgp), so I will do this in a follow-up PR to avoid unnecessary conflict resolution (A lot of lines need to be shortened to proper length - This would probably mess with everyhting).